great-idea/css/index.css

```
@@ -63,4 +63,111 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Your code starts here! */
+
+ body{
+	/* width:876px; 
+	margin:0 auto;  */
+	
+} 
+
+.master{
+	display:flex;
+	width: 100%;
+	justify-content: center;
+}
+
+.container{
+	display:flex;
+	flex-direction: column;
+	justify-items: center;
+	width: 876px;
+}
+
+header{
+	display: flex;
+	justify-content: space-around;
+	width:100%;
+	height: 10%;
+	margin-bottom: 80px;
+	align-items: center;
+	padding-top: 25px;
+}
+
+nav{
+	display: flex;
+	align-content: center;
+	justify-content: space-around;
+	align-content: baseline;
+	flex-grow: 2;
+}
+
+a{
+	text-decoration: none;
+	color:black
+}
+
+address p{
+	margin-bottom: 10px;
+}
+
+.large-items {
+	display: flex;
+	justify-content: space-around;
+	align-items: center; 
+	width: 100%;
+	margin-bottom: 150px;
+}
+
+.innovation{
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	height: 379px;
+	width: 379px;
+}
+.large-items div h1{
+	text-align: center;
+	font-size: 75px;
+	letter-spacing: 4px;
+	font-stretch: expanded;
+}
+
+.large-items div p{
+	height: 32px;
+	width: 175px;
+	padding-top: 5px;
+	border: 1px solid black;
+	text-align: center;
+	letter-spacing: 2px;
+}
+
+.Features-and-about{
+	display: flex;
+	width: 100%;
+	margin-bottom: 45px;
+	justify-content: center;
+}
+
```

```suggestion
h3 {
    margin-left: 7px;
}
```
Margin on h3 to align headers with paragraphs